i18n/en.json

```diff
@@ -4012,6 +4012,10 @@
     "translation": "Invalid per sec for rate limit settings.  Must be a positive number"
   },
   {
+    "id": "model.config.is_valid.go_routines.app_error",
+    "translation": "Invalid go routines threshold for rate limit settings.  Must be a positive number"
```

Comment:
"go routines" should be one word "goroutines" and should be singular here like `Invalid goroutine threshold...`